tools/sboms/generator/sbom_test.go

```diff
@@ -0,0 +1,122 @@
+package generator
+
+import (
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+
+	"github.com/sclevine/spec"
+	"github.com/sclevine/spec/report"
+
+	h "github.com/buildpacks/lifecycle/testhelpers"
+)
+
+func TestSbomGenerator(t *testing.T) {
+	spec.Run(t, "SbomGenerator", testSbomGenerator, spec.Report(report.Terminal{}))
+}
+
+func testSbomGenerator(t *testing.T, when spec.G, it spec.S) {
+	when("given a directory with a simple GO binary", func() {
+		var (
+			binaryPath string
+		)
+
+		it("generates an SBOM for that binary", func() {
+			var err error
+
+			tmpDir, err := os.MkdirTemp("", "sbom-test")
+			h.AssertNil(t, err)
+
+			tmpBinaryDir, err := os.MkdirTemp("", "sbom-test-binary")
+			h.AssertNil(t, err)
+
+			copyFile(t, tmpDir, "go.mod.template", "go.mod")
+			copyFile(t, tmpDir, "go.sum.template", "go.sum")
+			copyFile(t, tmpDir, "simple-app.go.template", "simple-app.go")
```

Comment:
Interesting use of `.template` ...I haven't seen it before, do we need it? It's common to call the directory with fixtures `testdata`

Reply:
ok. I have rewritten this code with the use of a new _testdata_ directory.